components/Slider/src/MDCSlider.m

```diff
@@ -227,7 +243,7 @@ - (CGSize)intrinsicContentSize {
 }
 
 - (BOOL)pointInside:(CGPoint)point withEvent:(__unused UIEvent *)event {
-  CGFloat dx = MIN(0, kSliderThumbRadius - kSliderMinTouchSize / 2);
+  CGFloat dx = MIN(0, _thumbTrack.thumbRadius - kSliderMinTouchSize / 2);
```

Comment:
Nit: Prefer `self.thumbRadius` to avoid depending on this implementation detail in the future.

Reply:
Done.